app/views/home/dashboard.html.erb

```diff
@@ -26,8 +26,8 @@
       </header>
       <div class="panel-body">
         <ul>
-          <% Project.order(created_at: :desc).take(5).each do |idea| %>
-            <li><%= link_to idea.name, idea %></li>
+          <% Project.joins(:project_votes).select('`projects`.id, `projects`.name, count(*) as count').group('`projects`.id').order('count DESC').take(5).each do |project| %>
+            <li><%= link_to project.name, project %></li>
           <% end %>
```

Comment:
It would be a good practice to move the model logic out of the view and put it in a scope and then use the scope in the controller and view will only have an instance variable.  The same would apply to the welcome view.